### PR TITLE
STREAMS-1528 - Add documentation about new webhook testing endpoint 

### PR DESCRIPTION
**Pull request by guillaumelamirand**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

add documentation about new test endpoint for webhook and kafka subscribers.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Does not expose any sensitive information
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links

### CODE REVIEW

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 1]
```diff
@@ -78,6 +78,24 @@ List of HTTP status codes that can be returned when trying to get a kafka subscr
 | 200 Ok | Indicates that the subscription requested is valid and has been retrieved. |
 | 404 Not found | Indicates that the requested URL or subscription requested does not exist. |
 
+## Testing a kafka subscription
+
+You can test a kafka subscription by making an HTTP Post request on the following endpoint:
+
+`POST /subscribers/webhook/subscriptions/{subscriptionId}/test`
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-kafka.md:85]: This section is related to Kafka subscription but the endpoint is pointing to /subscribers/webhook ?

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 2]
```diff
@@ -78,6 +78,24 @@ List of HTTP status codes that can be returned when trying to get a kafka subscr
...
+
+The body could contain any JSON object and will be sent as is to the subscription identified.
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-kafka.md:87]:
```suggestion
The request body can contain any JSON object and will be sent as is to the identified subscription.
```

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 3]
```diff
@@ -78,6 +78,24 @@ List of HTTP status codes that can be returned when trying to get a kafka subscr
...
+
+### Test status codes
+
+Below the list of HTTP status codes that can be returned when trying to test a kafka subscription:
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-kafka.md:91]:
```suggestion
Below the list of HTTP status codes that can be returned while testing a Kafka subscription:
```

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 4]
```diff
@@ -78,6 +78,24 @@ List of HTTP status codes that can be returned when trying to get a kafka subscr
 | 200 Ok | Indicates that the subscription requested is valid and has been retrieved. |
 | 404 Not found | Indicates that the requested URL or subscription requested does not exist. |
 
+## Testing a kafka subscription
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-kafka.md:81]:
```suggestion
## Testing a Kafka subscription
```

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 5]
```diff
@@ -78,6 +78,24 @@ List of HTTP status codes that can be returned when trying to get a kafka subscr
...
+
+You can test a kafka subscription by making an HTTP Post request on the following endpoint:
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-kafka.md:83]:
```suggestion
You can test a Kafka subscription by making an HTTP Post request on the following endpoint:
```

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 6]
```diff
@@ -78,6 +78,24 @@ List of HTTP status codes that can be returned when trying to get a kafka subscr
...
+
+| Code | Comment |
+|------|---------|
+| 202 Accepted | Indicates that the payload has been successufuly sent to the subscription. |
+| 400 Bad Request | Indicates that the provided data are invalid. |
+| 404 Not found | Indicates that the requested URL does not exist. |
+
 ## Getting kafka subscriptions for a topic
 
 In order to get existing subscriptions, simply do the following GET request on your topic:
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-kafka.md:101]:
```suggestion
In order to get existing subscriptions, just do the following `GET` request on your topic:
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 7]
```diff
@@ -76,6 +76,24 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
 | 200 Ok | Indicates that the subscription requested is valid and has been retrieved. |
 | 404 Not found | Indicates that the requested URL or subscription requested does not exist. |
 
+## Testing a webhook subscription
+
+You can test a webhook subscription by making an HTTP Post request on the following endpoint:
+
+`POST /subscribers/webhook/subscriptions/{subscriptionId}/test`
+
+The body could contain any JSON object and will be sent as is to the subscription identified.
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-webhook.md:85]:
```suggestion
The request body can contain any JSON object and will be sent as is to the identified subscription.
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 8]
```diff
@@ -76,6 +76,24 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
...
+
+### Test status codes
+
+Below the list of HTTP status codes that can be returned when trying to test a webhook subscription:
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-webhook.md:89]:
```suggestion
Below the list of HTTP status codes that can be returned while testing a webhook subscription:
```

**Review by allandenis38 — CHANGES_REQUESTED** [review]

File: content/en/docs/subscribers/subscriber-kafka.md (hunk 4 above)
Comment (guillaumelamirand) [reply content/en/docs/subscribers/subscriber-kafka.md:81]: There is no majuscul for the section.

- Getting a kafka subscription
-  topping a kafka subscription

File: content/en/docs/subscribers/subscriber-kafka.md (hunk 5 above)
Comment (guillaumelamirand) [reply content/en/docs/subscribers/subscriber-kafka.md:83]: Same as above, no maj on others sections

**Review by allandenis38 — APPROVED** [review]

**Comment by allandenis38** [comment]
@lbadenhop PR is ready for your review
Preview: https://deploy-preview-54--streams-open-docs.netlify.app/

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 9]
```diff
@@ -78,9 +78,27 @@ List of HTTP status codes that can be returned when trying to get a kafka subscr
 | 200 Ok | Indicates that the subscription requested is valid and has been retrieved. |
 | 404 Not found | Indicates that the requested URL or subscription requested does not exist. |
 
+## Testing a kafka subscription
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-kafka.md:81]:
```suggestion
## Testing a Kafka subscription
```

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 10]
```diff
@@ -78,9 +78,27 @@ List of HTTP status codes that can be returned when trying to get a kafka subscr
...
+
+You can test a kafka subscription by making an HTTP Post request on the following endpoint:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-kafka.md:83]:
```suggestion
You can test a Kafka subscription by making an HTTP POST request on the following endpoint:
```

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 11]
```diff
@@ -78,9 +78,27 @@ List of HTTP status codes that can be returned when trying to get a kafka subscr
...
+
+`POST /subscribers/kafka/subscriptions/{subscriptionId}/test`
+
+The request body can contain any JSON object and will be sent as is to the identified subscription.
+
+### Test status codes
+
+Below the list of HTTP status codes that can be returned while testing a Kafka subscription:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-kafka.md:91]:
```suggestion
The following HTTP status codes can be returned while testing a Kafka subscription:
```

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 12]
```diff
@@ -78,9 +78,27 @@ List of HTTP status codes that can be returned when trying to get a kafka subscr
...
+| Code | Comment |
+|------|---------|
+| 202 Accepted | Indicates that the payload has been successufuly sent to the subscription. |
+| 400 Bad Request | Indicates that the provided data are invalid. |
+| 404 Not found | Indicates that the requested URL does not exist. |
+
 ## Getting kafka subscriptions for a topic
 
-In order to get existing subscriptions, simply do the following GET request on your topic:
+In order to get existing subscriptions, just do the following `GET` request on your topic:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-kafka.md:101]:
```suggestion
Use the following `GET` request on your topic to get existing subscriptions:
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 13]
```diff
@@ -76,6 +76,24 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
...
+
+You can test a webhook subscription by making an HTTP Post request on the following endpoint:
+
+`POST /subscribers/webhook/subscriptions/{subscriptionId}/test`
+
+The request body can contain any JSON object and will be sent as is to the identified subscription.
+
+### Test status codes
+
+Below the list of HTTP status codes that can be returned while testing a webhook subscription:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-webhook.md:89]:
```suggestion
The following HTTP status codes can be returned while testing a webhook subscription:
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 14]
```diff
@@ -76,6 +76,24 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
...
+
+| Code | Comment |
+|------|---------|
+| 202 Accepted | Indicates that the payload has been successufuly sent to the subscription. |
+| 400 Bad Request | Indicates that the provided data are invalid. |
+| 404 Not found | Indicates that the requested URL does not exist. |
+
 ## Getting webhook subscriptions for a topic
 
 In order to get existing subscriptions, simply do the following GET request on your topic:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-webhook.md:99]:
```suggestion
To get existing subscriptions, do the following GET request on your topic:
```

**Review by lbadenhop — COMMENTED** [review]
See suggested comments

**Pull request merged**